app/src/main/java/com/example/bookwormadventuresdeluxe2/MyProfileFragment.java

```diff
@@ -130,4 +162,10 @@ public void onClick(View view)
             Log.d(TAG, e.getMessage());
         }
     }
+
+    @Override
+    public void onCallback(UserObject userObject)
+    {
+
```

Comment:
What is the purpose of this?

Reply:
firebase User Auth is also called User

Reply:
userprofile would be a better name yea

Reply:
that is for passing an object onsuccess, so it doesnt trigger with null